Training/Program.cs

```diff
@@ -1,7 +1,77 @@
+using Training;
+
 namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   class MyList<T> {
+      T[] mVariables;
+      public MyList () => mVariables = new T[4];
+
+      public int Count => mSize;
+
+      public int Capacity => mVariables.Length;
+
+      public T this[int index] { get { return mVariables[index]; } set { mVariables[index] = value; } }
+
+      public void Add (T element) {
+         if (Count == Capacity) Array.Resize (ref mVariables, Capacity * 2);
+         mVariables[mSize++] = element;
+      }
+
+      public bool Remove (T element) {
+         if (mVariables.Contains (element)) {

```

Comment:
Store 'mVariables.Contains (element)' in a local var as it is used more than a time.